relay-server/src/actors/outcome.rs

```diff
@@ -83,6 +83,8 @@ pub struct TrackOutcome {
     pub remote_addr: Option<IpAddr>,
     /// The event's data category.
     pub category: DataCategory,
+    /// The number of events or total attachment size in bytes.
+    pub quantity: usize,
```

Comment:
not sure if this is a huge deal but we limit an individual outcome's quantity to 32 bits, it seems very unlikely a large attachment would ever make it in but we could make this a `u32` to be consistent

Reply:
Good point, @RyanSkonnord and I discussed this yesterday offline. We chose `usize`, as this matches what `EnvelopeSummary` does and is more ergonomic to work with given that we derive this number from memory slices.

There are validation functions earlier in the pipeline that ensure attachments and entire requests never exceed an even lower size limit than what `u32` permits. Precisely, this limit is _100MiB_. 

I'm open to changing this in all places if you prefer.

Reply:
ok gotcha! yeah keeping as is sounds good to me.